### ISSUE TITLE
Examples for expressions references & some extra info

### ISSUE THREAD

**Issue by ZeroAlvein** [GDevelopApp/GDevelop-documentation#240]
So here https://wiki.gdevelop.io/gdevelop5/all-features/expressions-reference/

In most cases it just say what it is but does not explain what it is
Or worse how it works 

For example abs()
What is abs?

I know you know but does new user seeing this expression for the 1st time knows?
And all such user see is
abs() Absolute value

What means absolute?
English is not my native language
I did not have idea what it means

If it would be like
abs() Absolute value

Using abs() on any number will return it without minus - 
If its negative number
For example abs(-20) will return 20
Useful for counting down

If we set variable to -20 and we gonna add 1 to it
We will get -20 > -19 > -18

If we use abs(variable) to display it in text object
Then we will see 20 > 19 > 18
Which will be also useful for making countdown timers

Stuff like that is needed on this page
I could write examples with descriptions for some expressions since i don't understand all of them myself
But any user who would see proper description and practical usage would not need additional explanations
And it could help deduce other stuff by user on its own

Problem is i have dyslexia and my engrish and my grammar
Someone would need to review everything i write

So before i gonna spam here with such examples/descriptions
I would want to know would it have any sense
Cause i would also want to provide such info for extensions
And some other documentations which kinda need it

**Comment by D8H** [comment]
Indeed, "absolute value" is Math jargon and the description is the same as the name. The description could be changed to "Return the non-negative value by removing the sign. The absolute value of −8 is 8."

**Comment by D8H** [comment]
@ZeroAlvein Do you have other actions, conditions or expressions in mind that could use more explanations?
